app/controllers/provider_foreman_controller.rb

```diff
@@ -905,7 +864,7 @@ def configuration_profile_summary_tab_selected?
   def construct_edit
     @edit ||= {}
     @edit[:current] = {:name       => @provider_cfgmgmt.name,
-                       :provtype   => model_to_name(@provider_cfgmgmt.type),
+                       :provtype   => @provider_cfgmgmt.provider_class.description,
```

Comment:
@bdunne - it is a provider - would you be able to point where it is an ExtManagementSystem to correct that?